package/kedro_viz/models/graph.py

```diff
@@ -27,6 +27,7 @@
 # limitations under the License.
 """`kedro_viz.models.graph` defines data models to represent Kedro entities in a viz graph."""
 # pylint: disable=protected-access
+# pylint: disable=logging-fstring-interpolation

```

Comment:
This is actually a good linting error. We shouldn't disable it.

Reply:
Yes, I will remove this as I am also no longer using this.